crates/solidity/outputs/cargo/crate/src/generated/support/optional_helper.rs

```diff
@@ -0,0 +1,19 @@
+// This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+use super::parser_result::ParserResult;
+
+pub struct OptionalHelper;
+
+impl OptionalHelper {
```

Comment:
Love the refactoring of these outside the giant parser code!
I wonder why are these generated though? can they be moved (perhaps later) to the hand-authored modules instead? We discussed minimizing the generated code as much as we can, to make it easier for us to write user-facing APIs, and different algorithms, directly in Rust instead of templates.

Reply:
These aren't generated per se - they are copied from the parser_runtime crate.